samples/js-chatbot/genkit-app/package.json

```diff
@@ -26,7 +26,7 @@
     "rxjs": "~7.8.0",
     "tslib": "^2.3.0",
     "zone.js": "~0.14.3",
-    "genkit": "^1.0.0-rc.15"
+    "genkit": "^1.0.0-rc.17"
```

Comment:
technically it's not necessary, it will automatically pick up the latest rc:
https://jubianchi.github.io/semver-check/#/^1.0.0-rc.12/1.0.0-rc.17

and even full 1.0 release
https://jubianchi.github.io/semver-check/#/^1.0.0-rc.12/1.0.4